npg_qc_viewer/t/80-app-run.t

```diff
@@ -227,7 +233,83 @@ subtest 'Test for run + lane + plexes' => sub {
                     qr/Use of uninitialized value \$id in exists/, ],
                       'Expected warning for run folder found';  
   }
-}
+};
+
+subtest 'Tag metrics as first check in summary table' =>  sub {
```

Comment:
Too much test data. Please create a test with as many tags as necessary to illustrate your test case. 3-4 tags should be enough.